queue/src/main/java/org/killbill/queue/dao/QueueSqlDao.java

```diff
@@ -75,6 +75,7 @@ long getNbReadyEntries(@Bind("now") Date now,
     List<T> getEntriesLeftBehind(@Bind("max") int max,
                                  @Bind("now") Date now,
                                  @Bind("reapingDate") Date reapingDate,
+                                 @Bind("owner") String owner,
```

Comment:
I don't think `owner` is used since we fetch all entries and then have the `DBBackedQueue#reapEntries` determine which one are *others* versus *us*.

Reply:
Good catch - it was let over from an initial implementation. See 6279fe9dc0663ef7fcb7d6e3f00cf99bdf4999da.